{{ cookiecutter.project_name }}/{{ cookiecutter.package_name }}/scripts/train.py

```diff
@@ -22,7 +22,16 @@
     "config_file", type=click.Path(exists=True), default="/mnt/configs/config.yml"
 )
 @setup_logging_env
-def main(config_file="config.yml"):
+def _main(config_file):
+    """
+    This pattern aids in writing tests, as the function with the click decorators can be
```

Comment:
I don't know if this doc string is necessary. Especially about writing tests. This is also what shows when you do.
`python train.py --help`

"""Train CLI entrypoint"""

While the doc string on `def main()` is more specific.
"""